pkg/manifests/manifests.go

```diff
@@ -78,6 +79,31 @@ const (
 	CustomResourceDefinitionManifest = "manifests/00-custom-resource-definition.yaml"
 )
 
+// Copy needed files from the manifests/ directory.  This is necessary in order
+// to embed these files here because manifests/ is a top-level directory in the
+// source tree, and go:embed does not allow paths containing "../".
+//
+//go:generate mkdir -p manifests
+//go:generate cp ../../manifests/00-namespace.yaml manifests/
+//go:generate cp ../../manifests/00-custom-resource-definition.yaml manifests/
```

Comment:
A couple of questions.  Why are these two files the only ones needed from the manifests/ directory?  And why not permanently move them to assets/  instead of creating a temporary directory?

Reply:
> Then they wouldn't be in the right place for CVO to read them. The manifests still need to be installed by CVO if the cluster admin doesn't configure them with the installer.

Seems like a worthwhile comment to add so that somebody doesn't move things in the future.

Reply:
https://github.com/openshift/cluster-ingress-operator/compare/62697d237725ec70f45138087875c3e644b901f1..3dafe96f29556f24a6cfdcd199b669681c83221b adds comments.